src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -138,6 +138,18 @@ class CallOpcodesSpec extends WordSpec with Matchers {
         call.stateOut.stack.pop._1 shouldEqual UInt256.One
       }
 
+      "should store contract's return data in memory" in {
+        //here the passed data size is equal to the contract's return data size
+
+        val expectedData = fxt.inputData.take(fxt.inputData.size / 2)
```

Comment:
For me it wasn't matched immediately (on a quick PR review for example) with the comment you point out. If you think it's self explanatory, no need to do anything.